### PR TITLE
[Ide] Initial support for deleting welcome page items.

### PR DESCRIPTION
**Pull request by Therzok**
This is WIP.

@vancura I need colors for text and disabled variants for any icon we show in the recent projects list - md-solution, md-workspace?

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs [hunk 1]
```diff
@@ -115,7 +117,9 @@ public static void LoadStyles ()
 			{
 				var bgColor = Gui.Styles.BaseBackgroundColor.ToHexString (false);
 				var fgColor = Gui.Styles.BaseForegroundColor.ToHexString (false);
+				var fgDisabledColor = Gui.Styles.BaseForegroundColor.WithAlpha (0.7).ToHexString (false);
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs]: This is a NOP, we need real hex values.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs [hunk 2]
```diff
@@ -92,7 +93,7 @@ public static class SolutionTile
 						public static string HoverBorderColor { get; internal set; }
 						public const int TitleFontSize = 12;
 						public const int PathFontSize = 10;
-						public const int TextLeftPadding = 38;
+						public const int TextLeftPadding = 38 - 14;
```
Comment (vancura) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs]: @Therzok: I can see this issue now, will fix in the following commit, sorry.

Comment (vancura) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs]: I mean, I should use `HorizontalRepadding` there.

Comment (vancura) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.WelcomePage/Style.cs]: Fixed in https://github.com/mono/monodevelop/pull/1525/commits/1ea81fb3236cc19d8bd47fb8fc5f16d155f57465

**Comment by vancura** [comment]
Obsolete, closing.

**Pull request closed without merging**